ui/src/pages/Forbidden.vue

```diff
@@ -1,6 +1,31 @@
 <template>
-  <div class="fixed-center text-center">
-    <p class="q-my-lg text-faded">Ops, você não tem permissão para acessar esta página <strong>(403).</strong></p>
-    <q-btn color="primary" flat @click="$router.push('/')">Voltar</q-btn>
+  <div class="bg-grey-2 column forbidden-error-page items-center justify-around">
+    <div class="column full-width items-center justify-center q-gutter-md q-py-md row">
+      <q-img src="../assets/gear.svg" :width="gearSize" />
+      <div class="text-bold text-h1 text-primary">403</div>
+      <div class="text-bold text-center text-h6 text-grey-9">Ops, parece que você não tem acesso para este recurso!</div>
+      <qas-btn class="text-bold text-primary" label="Voltar ao início" @click="$router.push('/')" />
```

Comment:
a prop `to` não resolve? pq precisa do @click?

Reply:
sim, corrigido para o uso do `:to`